src/app/my-movies/my-movies.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+
+@Component({
+  selector: 'app-my-movies',
+  templateUrl: './my-movies.component.html',
+  styleUrls: ['./my-movies.component.css']
+})
+export class MyMoviesComponent implements OnInit {
+
+  message = new TemplateMessage();
+  sorts = ['- - -', 'Najstarsze filmy', 'Ostatnio dodane filmy'];
+
+  constructor() { }
+
+  ngOnInit() { }
+
+}
+
+class TemplateMessage {
```

Comment:
nie widzę żadnego powodu żeby to była klasa - wartość sorta może być zwykłym stringiem